modules/locate.js

```diff
@@ -0,0 +1,130 @@
+const rp = require("request-promise-native");
+const Discord = require("discord.js");
+const log = require("log4js").getLogger("standard");
+
+class Locate{
+
+    constructor(){
+        this.geocoder = 'https://maps.googleapis.com/maps/api/geocode/json?new_forward_geocoder=true&key='+process.env.GOOGLE_TOKEN+'&address=';
+        this.googleStaticMap = 'https://maps.googleapis.com/maps/api/staticmap?autoscale=1&size=640x320&maptype=' +
+            'roadmap&format=png&visual_refresh=true&key='+process.env.GOOGLE_STATIC_TOKEN;
+        this.googleStaticMapStandardMarkup = '&markers=size:large|color:0x513dc2|';
+        this.wizardsLocator = 'http://locator.wizards.com/Service/LocationService.svc/GetLocations';
+        this.wizardsStoreUrl = 'http://locator.wizards.com/#brand=magic&a=location&massmarket=no&';
+        this.commands = {
+            locate: {
+                aliases: [],
+                inline: false,
+                description: "Lists stores that are nearby the specified location",
+                help: '',
+                examples: ["!locate New York"]
+            }
+        };
+        log.info("Locate ready");
+    }
+
+    getCommands(){
+        return this.commands;
+    }
+
+    generateRequestBody(geometry){
+        let requestBody = {
+            count: 9,
+            filter_mass_markets: true,
+            language: "en-us",
+            page: 1,
+            request:{
+                EarliestEventStartDate: null,
+                LatestEventStartDate: null,
+                LocalTime: '/Date('+Date.now()+')/',
+                EventTypeCodes: [],
+                PlayFormatCodes: [],
+                MarketingProgramCodes: [],
+                ProductLineCodes: ["MG"],
+                SalesBrandCodes: ["MG"]
+            }
+        };
+        if(geometry.bounds){
+            requestBody.request.North = geometry.bounds.northeast.lat;
+            requestBody.request.East = geometry.bounds.northeast.lng;
+            requestBody.request.South = geometry.bounds.southwest.lat;
+            requestBody.request.West = geometry.bounds.southwest.lng;
+        }else{
+            requestBody.request.North = geometry.location.lat;
+            requestBody.request.East = geometry.location.lng;
+            requestBody.request.South = geometry.location.lat;
+            requestBody.request.West = geometry.location.lng;
+        }
+        return requestBody;
+    }
+
+    generateEmbed(results,googleResult){
+        const descriptions = [];
+        const googleStaticMap = [this.googleStaticMap];
+        results.forEach(result=>{
+            const description = {name:["*",(results.indexOf(result)+1),": ",result.Address.Name,"*","\n"].join(""),
```

Comment:
I'd prefer to have an ES6 template here instead of String-`join()`ing. 
```
{name: `*${results.indexOf(result)+1}: ${result.Address.Name}*\n`, inline: true}
```